libtsuba/src/RDGCore.cpp

```diff
@@ -185,7 +185,8 @@ katana::RDGCore::AddNodeProperties(
   auto written_prop_names = KATANA_CHECKED(AddProperties(
       props, &node_properties_, &part_header_.node_prop_info_list()));
   // store write properties into transaction context
-  txn_ctx->InsertNodePropertyWrite<std::set<std::string>>(written_prop_names);
+  txn_ctx->InsertNodePropertyWrite<std::set<std::string>>(
+      rdg_dir_.string(), written_prop_names);
```

Comment:
@ddn0, this is how a lot of people approach URI::string, which is just give me the string version of this file location.  I don't think they are expecting percent escapes. (This is a reference to Donald's #842.

Reply:
As long as we go through tsuba to read the paths, it should be fine. The string version is a proper URI, the local fs backend knows how to decode URIs.

Reply:
Ah, that is ok with me.